Squeezer/src/main/java/uk/org/ngo/squeezer/itemlist/AlarmsActivity.java

```diff
@@ -181,20 +181,20 @@ protected void orderPage(@NonNull ISqueezeService service, int start) {
 
         @Override
         public void onItemsReceived(final int count, final int start, Map<String, String> parameters, final List<AlarmPlaylist> items, Class<AlarmPlaylist> dataType) {
-            if (start == 0) {
-                mAlarmPlaylists.clear();
-            }
+            getUIThreadHandler().post(new Runnable() {
+                @Override
+                public void run() {
+                    if (start == 0) {
+                        mAlarmPlaylists.clear();
+                    }
 
-            mAlarmPlaylists.addAll(items);
-            if (start + items.size() >= count) {
-                getUIThreadHandler().post(new Runnable() {
-                    @Override
-                    public void run() {
+                    mAlarmPlaylists.addAll(items);
+                    if (start + items.size() >= count) {
                         mAlarmView.setAlarmPlaylists(mAlarmPlaylists);
```

Comment:
I'll merge as is for the moment, but what if just this line was changed to be:

```
mAlarmView.setAlarmPlaylists(ImmutableList.copyOf(mAlarmPlaylists);
```
instead?